tests/sandbox/sandbox_gov.lua

```diff
@@ -0,0 +1,60 @@
+-- Copyright (c) Microsoft Corporation. All rights reserved.
+-- Licensed under the Apache 2.0 License.
+
+-- This file defines the default initial contents (ie, Lua scripts) of the governance scripts table.
+return {
+  pass = [[
+  tables, calls, votes = ...
+
+  -- interface definitions
+  PASSED = 1
+  PENDING = 0
+  REJECTED = -1
+
```

Comment:
A comment similar to:

The constitution in a real CCF application deployment would at least count votes and compare to a threshold of members, but in the sandbox sample, all votes pass automatically.